portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,35 @@
 package com.google.sps.servlets;
 
+
 import java.io.IOException;
+import java.util.ArrayList;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+  String[] listOfFacts = {"Fact #1: I'm a vegetarian.", 
+  "Fact #2: My hobbies include jailbreaking/modding, crochet, and playing video games.",
+  "Fact #3: I speak French (poorly)."
+};
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Allie!</h1>");
+    String json = convertToJsonUsingGson(listOfFacts);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJsonUsingGson(String[] stringList) {
```

Comment:
"Helpers not holders"

You want abstractions that help encapsulate complexity, not move complexity from one place to another. IMO `convertToJsonUsingGson` doesn't provide much value as an abstraction. The name gives away what the abstraction is trying to encapsulate. My preference would be to delete this function and just use Gson() directly within the function (I don't think it's being called anywhere else). Another option would be to rename it so it doesn't have `Gson` on it's name (the caller to this function just cares that the output is json, not that Gson was used).